src/main/kotlin/no/nav/tiltakspenger/arena/repository/SakRepository.kt

```diff
@@ -14,19 +14,19 @@ Vi har også endret så vi bruker mer enums, og så utfall inkluderes vedtakene.
 
 Saker som returneres i liste:
 
-Ytelsen må inneholde vedtak med fra_dato <= tom-dato i input. Dersom tom-dato er blank vil 31.12.2099 benyttes.
+Ytelsen må inneholde vedtak med fra_dato <= tom-dato i input. Dersom tom-dato er blank vil 31.12.2299 benyttes.
 Ytelsen må inneholde vedtak med til_dato >= fom-dato i input. Dersom fom-dato er blank vil 01.01.1900 benyttes.
-Ytelsen må inneholde vedtak som oppfyller et av følgende to alternativer:
-Vedtakstype er Ny rettighet (O), Endring (E) eller Gjenopptak (G), samt at utfallskode ikke er Avbrutt (AVBRUTT) eller Nei (NEI).
-Vedtakstype er Ny rettighet (O), samt at utfallskode er Nei (NEI).
+Ytelsen må inneholde vedtak hvor:
+- typen må være Ny rettighet (O), Endring (E) eller Gjenopptak (G)
+- utfallskode må ikke være Avbrutt (AVBRUTT) eller Nei (NEI)
+Ytelsen må ha minst ett vedtak som oppfyller kravene for å returneres,
+og bare vedtakene som oppfyller kravene inkluderes
 Ytelsen returneres ikke dersom sakstatus er Historisert (HIST).
-Alle vedtak i de returnerte sakene listes, med unntak av ytelsesvedtak av type "§11-5 Nedsatt arbeidsevne"(AA115) og avbrutte vedtak.
```

Comment:
Veldig bra du har oppdatert kravene for ytelsen! 🙌 
Er dette caser som testes på i koden? Og returneres det i så fall noen feilmelding til brukere av tiltakspenger-arena som også informerer om dette? Eller er dette bare noe som er kjekt å vite om man er i koden og titter? Og at disse 'kravene’ er noe man må ha i mente om man skal skrive ny kode som henter data fra Arena?

Reply:
Mest det siste, man henter ut bare et subsett av sakene og vedtakene. Om jeg tester godt nok på det i koden får nesten du avgjøre, men litt er det.. :)